test/plugin_functional/test_suites/core_plugins/rendering.ts

```diff
@@ -106,7 +106,7 @@ export default function({ getService, getPageObjects }: PluginFunctionalProvider
     });
 
     it('renders "legacy" application', async () => {
-      await navigateTo('/render/core_plugin_legacy');
+      await navigateTo('/render/legacy_app');
```

Comment:
This whole test file is currently skipped due to flakiness, but I went ahead and fixed this test. It was incorrectly rendering an application on a path that was not it's appRoute, causing ScopedHistory to throw. This is now just a regular legacy angular application.